amd/src/mutations.js

```diff
@@ -217,9 +232,31 @@ export default class {
                     boardid: state.board.id,
                     timestamp: state.common.timestamp,
                 },
+                fail: () => {
+                    this.processUpdateFail();
+                },
             }])[0];
 
             this.processUpdates(stateManager, result);
+            this.resetUpdateFails();
+        }
+    }
+
+    /**
+     * Reset update fails.
+     */
+    resetUpdateFails() {
+        this.updateFails = 0;
+        document.querySelector('.mod_kanban_update_error').classList.add('hidden');
```

Comment:
you probably want to add this to the selectors.js and use the constant defined there?

Also AFAIK CSS classes should only be used for styling. For "queryselectoring" an element a data attribute should be used.

Reply:
That would mean a lot of changes in the plugin (as it is done in many other places). Would suggest to move that to another issue.